<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="vec1xEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vec1yEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vec2xEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vec2yEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set1Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set2Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print2Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QLabel" name="array2Label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="max1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sum1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>sum</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
